src/Microsoft.Dnx.Host.Clr/project.json

```diff
@@ -1,8 +1,14 @@
 {
-    "version": "1.0.0-*",
-    "compilationOptions": { "define": [ "TRACE" ], "allowUnsafe": true, "warningsAsErrors": true, "keyFile": "KeyFile.snk" },
-    "dependencies": {
-    },
+  "version": "1.0.0-*",
+  "compilationOptions": {
+    "define": [ "TRACE" ],
+    "allowUnsafe": true,
+    "warningsAsErrors": true,
+    "keyFile": "KeyFile.snk"
+  },
+  "dependencies": {
+    "Microsoft.Dnx.Runtime.Json.Sources": { "version": "1.0.0-*", "type": "build" }
```

Comment:
Why do we need it here? Does it work at all? Microsoft.Dnx.Host.Clr has a real strong name and cannot depend on non-signed assemblies (it may fail in the second build). This is the reason why it does not have any dependencies.